build.gradle.kts

```
@@ -9,11 +9,25 @@ plugins {
     alias(libs.plugins.protobuf)
     alias(libs.plugins.shadow.jar)
     alias(libs.plugins.kotlin.serialization)
+    alias(libs.plugins.git.versioning)
     application
 }
 
 group = "se.uulm.snowballr.backend"
-version = "0.1.0"
+version = "0.0.0"
+
+gitVersioning.apply {
+    refs {
+        tag("v(?<version>.*)") {
```

```suggestion
        tag("v(?<version>\d+.\d+.\d+)") {
```
Isn't this the more robust regex? Because theoretically somebody could accidentally create the tag "vollcool" or "v1.0" and then we would create the version "ollcool" or "1.0" which would be (totally) correct. What do you think?

---

Fine for me.